eidpdata/src/main/java/uk/ac/ucl/eidp/data/PoolStrategy.java

```diff
@@ -78,7 +79,9 @@ private void initialiseConnection() {
 
     @Override
     public List<Map<String, String>> processDbCall(String methodPath, Map<String, String> parameters) {
-        if (null == connection) initialiseConnection();
+        if (null == connection) {
+            initialiseConnection();
```

Comment:
\[Checkstyle\] WARNING: 'if' child have incorrect indentation level 12, expected level should be 6\.